include/miopen/miopen.h

```diff
@@ -4950,6 +4950,129 @@ MIOPEN_EXPORT miopenStatus_t miopenCTCLoss(miopenHandle_t handle,
+ * @param tDesc                    Tensor descriptor for target tensor (input)
+ * @param t                        Data tensor target (input)
+ * @param oDesc                    Tensor descriptor for output tensor (input)
+ * @param o                        Data tensor output (output)
+ * @param margin                   Margin (input)
+ * @param divisor                  Divisor (input)
+ * @return                         miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenHingeEmbeddingLossForward(miopenHandle_t handle,
+                                                             void* workspace,
+                                                             size_t workspaceSizeInBytes,
+                                                             miopenTensorDescriptor_t iDesc,
+                                                             const void* i,
+                                                             miopenTensorDescriptor_t tDesc,
+                                                             const void* t,
+                                                             miopenTensorDescriptor_t oDesc,
+                                                             void* o,
+                                                             float margin,
+                                                             float divisor);
+
+/*! @brief Execute a HingeEmbeddingLoss backward layer
+ *
+ * @param handle                   MIOpen handle (input)
+ * @param iDesc                    Tensor descriptor for input tensor (input)
+ * @param i                        Data tensor input (input)
+ * @param tDesc                    Tensor descriptor for target tensor (input)
+ * @param t                        Data tensor target (input)
+ * @param doDesc                   Tensor descriptor for output gradient (input)
+ * @param dO                       Gradient of output (input)
+ * @param diDesc                   Tensor descriptor for input gradient (input)
+ * @param dI                       Gradient of input (output)
+ * @param dtDesc                   Tensor descriptor for target gradient (input)
+ * @param dT                       Gradient of target (output)
+ * @param margin                   Margin (input)
+ * @param divisor                  Divisor (input)
+ * @return                         miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenHingeEmbeddingLossBackward(miopenHandle_t handle,
+                                                              miopenTensorDescriptor_t iDesc,
+                                                              const void* i,
+                                                              miopenTensorDescriptor_t tDesc,
+                                                              const void* t,
+                                                              miopenTensorDescriptor_t doDesc,
+                                                              const void* dO,
```

Comment:
There doesn't seem to be a need to use capital letters for dO. I looked at other functions in the same file, and it seems more common to use dout, doutput, and dy.

Reply:
I have updated variable name: i => input, t => target, o => output, dO => doutput, dI => dinput